### PR TITLE
Contributing and testing fork

### PR DESCRIPTION
**Pull request by ClaireDMT**
# PR
### What type of PR is this? (check all applicable)

- [ ]  Refactor
- [ ] New Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation
- [ ] Configuration
- [ ] Tests

### Linked Issues

Closes https://github.com/Contributees/First-Ruby-Quest/issues/33

## Feature Overview

Add a CONTRIBUTING.md to allow people to contribute, install the project and define what we expect from them.

### Challenges
NA
### Solutions
NA
## Tasks
- [x] add `CONTRIBUTING.md`
- [x] link README.md to the new `CONTRIBUTING.md`
- [x] add `.github/workflows/pull_request_template.md` to add a default PR description template
- [x] add a new `Procfile.front_dev` to have a process only with web server + css (without sidekiq and redis)

## New issues you found on the way
none

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # first-contributions
 Web app referencing ruby related open source issues suitable for a first contribution and/or junior developers (or any developpers!)
+
+
+## Contributing
+We warmly welcome everyone who is intersted in contributing. Please reference [contributing guidelines](CONTRIBUTING.md).
```
Comment (Oli0li) [inline README.md:6]:
```suggestion
We warmly welcome everyone who is interested in contributing. Please reference [contributing guidelines](CONTRIBUTING.md).
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+If you encounter any bugs or issues with **First-Ruby-Quest**, please open a new issue on GitHub. Provide as much detail as possible, including steps to reproduce the issue, expected behavior, and actual behavior. Also screenshots are welcome.
+
+### Feature Requests
+If you have ideas for new features or improvements, we'd love to hear them! Please open a new issue on GitHub to discuss your ideas and gather feedback from the community.
+
+
+### Submit a Pull Request
+
+1. Fork this repository on GitHub.
```
Comment (Oli0li) [inline CONTRIBUTING.md:30]: Maybe we could add something like this (feel free to reword if you prefer):

2. Clone that repository on your local machine.
If you haven't forked or cloned repositories before, please refer to [GitHub's help page](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo?tool=desktop)

To make it easier for people who don't know what forking and cloning is about.

**Review by Oli0li — CHANGES_REQUESTED** [review]
Thank you so much @ClaireDMT, I've left 2 comments but otherwise I find it very clear and welcoming!

**Comment by ClaireDMT** [comment]
thanks ! @Oli0li , i made the changes

**Review by Oli0li — APPROVED** [review]

**Pull request merged**